ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -245,6 +250,10 @@ public static void scrub(Map<String, String> attributes, ObjectNode json)
       processForBug1751753(json);
     }
 
+    if ("metrics".equals(docType)) {
```

Comment:
Indeed this is broad. App name is a concept that only exists in legacy telemetry (it's part of that URI structure), so not relevant for Glean. So namespace is what we'd need to use. It's a bit messy since each logical app has multiple namespaces (one per channel), but we can list out the ~6 values here explicitly.